### DISCUSSION TITLE
GitHub page not updating (repo shows newest commit)

### DISCUSSION THREAD

**Discussion by lukefretwell** [community/community#22226] [Other Feature Feedback, Questions, & Ideas]
<p>My son has updated his website (new RSVP link), but the commits aren’t showing on the actual GitHub-hosted web page.</p>
<ul>
<li>Website: <a href="https://narwhalcoding.com/" rel="nofollow noopener">https://narwhalcoding.com/</a>
</li>
<li>Repo: <a href="https://github.com/NarwhalCoding/narwhalcoding.github.io" rel="nofollow noopener">https://github.com/NarwhalCoding/narwhalcoding.github.io</a>
</li>
</ul>

**Comment by osama-mohamed** [comment, marked as answer]
<p>Github is now facing some errors on the database including github pages, read this for more information.</p>
<p><a href="https://blog.github.com/2018-10-21-october21-incident-report/" rel="nofollow noopener">https://blog.github.com/2018-10-21-october21-incident-report/</a></p>
<p><a href="https://status.github.com/messages" rel="nofollow noopener">https://status.github.com/messages</a></p>

**Comment by sophuric** [comment]
<p>It could be because of:</p>
<ul>
<li>The Jekyll will take delay to build</li>
</ul>
<p>or</p>
<ul>
<li>The cached page</li>
</ul>
<p>To clear the cache, goto your browser settings, and clear the history, it should clear the cache as well.</p>

**Comment by ghost** [comment]
<p>What I found is that Jekyll on GitHub may take several minutes before the site is updated and that the user gets no notification when the site has been updated.</p>
